Index: .idea/.gitignore
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
deleted file mode 100644
--- a/.idea/.gitignore	(revision c6cbe6220ae29fe10689504a95f046902f3ca632)
+++ /dev/null	(revision c6cbe6220ae29fe10689504a95f046902f3ca632)
@@ -1,5 +0,0 @@
-# Default ignored files
-/shelf/
-/workspace.xml
-# Editor-based HTTP Client requests
-/httpRequests/
